include/crocoddyl/core/activations/quadratic-barrier.hpp

```diff
@@ -40,19 +40,25 @@ struct ActivationBoundsTpl {
       throw_pretty("Invalid argument: "
                    << "The range of beta is between 0 and 1");
     }
-    using std::isfinite;
     for (std::size_t i = 0; i < static_cast<std::size_t>(lb.size()); ++i) {
       if (isfinite(lb(i)) && isfinite(ub(i))) {
```

Comment:
I wonder how this is able to compile if we remove `using std::isfinite`.

Reply:
Perhaps, we were not compiling this class in our codegen code (for benchmarking and unittesting)